modules/admin-ui/src/main/webapp/scripts/shared/partials/wizards/bulk-message.html

```diff
@@ -50,7 +50,7 @@
                          ng-change="wizard.step.applyTemplate()"
                          ng-model="wizard.step.ud.email_template"
                          ng-options="template as template.name for template in wizard.step.templates"
-                         data-placeholder="{{ 'EVENTS.SEND_MESSAGE.MESSAGE.EMAIL_TEMPLATE' | translate }}..."
+                         placeholder-text-single="'{{ 'EVENTS.SEND_MESSAGE.MESSAGE.EMAIL_TEMPLATE' | translate }}...'"
```

Comment:
Do you know where this is visible in the Admin-UI?

Reply:
@KatrinIhler That one is not visible as the bulk-message implementation was never finished.